test.html

```diff
@@ -0,0 +1,117 @@
+                <div class="wrapper">
+                    <div class="header__wrapper">
+                        <div class="header__logo">
+                            <a href="/" class="header__logo-link">
+                                <img src="./img/svg/KMarketer_Logo.svg" alt="We Take Care of the Future"
+                                    class="header__logo-pic">
+                            </a>
+                        </div>
+
+                        <div>
+                            <nav class="header_nav">
+                                <ul class="header__list">
+                                    <li class="header__item">
+                                        <a href="#!" class="header__link">Home</a>
+                                    </li>
+                                    <li class="header__item">
+                                        <a href="#!" class="header__link">Product</a>
+                                    </li>
+                                    <li class="header__item">
+                                        <a href="#!" class="header__link">Pricing</a>
+                                    </li>
+                                    <li class="header__item">
+                                        <a href="#!" class="header__link">Contact</a>
+                                    </li>
+                                    <ul class="header__icons">
+                                        <li>
+                                            <a href="#!" class="header__logo-link">
+                                                <img src="./img/svg/nav_bar_search_icon.svg" alt="search_icon"
+                                                    class="header__icons">
+                                            </a>
+                                        </li>
+                                        <li>
+                                            <a href="#!" class="header__logo-link">
+                                                <img src="./img/svg/nav_bar_shopping_cart_icon.svg"
+                                                    alt="shoppig_cart_icon" class="header__icons">
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </nav>
+                        </div>
+            </header>
+
+            <div class="main-container">
+                <h1>We Take Care of <br> the Future
+                </h1>
+                <h4>We know how large objects will act, but things on a <br>
+                    small scale just do not act that way.</h4>
+
+                <div class="buttons">
+                    <button class="button-get-quote">Get Quote Now</button>
+                    <button class="button-learn-more">Learn More</button>
+                </div>
+            </div>
+        </div>
+        </div>
+    </form>
+</body>
+
+</html>
+
+
+
+header__icons 
```

Comment:
seems like you forgot to remove these styles.

Reply:
@IvanCeban this is just test-file. Work-file is index.html